src/components.ts

```diff
@@ -312,7 +351,14 @@ export abstract class Component<T extends object = object> extends Controller<T>
     return (viewName) ?
       this.app.views.find(viewName, contextView.namespace) : contextView;
   }
-
+  /**
+   * Retrieve the appropriate view attribute's value for a given view instance
+   *
+   * @param attrName the name of the view attribute used with a component instance
+   * @returns any of the possible values that can be expressed with a view attribute
+   *
+   * @see https://derbyjs.com/docs/derby-0.10/views/template-syntax/view-attributes
```

Comment:
Pointing to the new GH Pages docs
```suggestion
   * @see https://derbyjs.github.io/derby/views/template-syntax/view-attributes
```